### PR TITLE
Flavio.126628867.react mfa

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you review please?

**Comment by flaviogranero** [comment]
👍 @chenkie @mpaktiti

**Comment by mpaktiti** [comment]
Merge sequence:
- https://github.com/auth0/docs/pull/1856
- https://github.com/auth0/docs/pull/1857
- https://github.com/auth0/docs/pull/1858

File: articles/client-platforms/_includes/_mfa-introduction.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Multi-factor Authentication (MFA) is an important method to add an extra security layer to your authentication flow. With this option enabled in your auth0 account, the process to grant user access to the application will require a verification step.  Besides the username and password combination, a verification code generated by a mobile application or sent by SMS is required. Currently auth0 supports [auth0 Guardian](/multifactor-authentication/guardian), Google Authenticator and Duo. Sending the code as SMS is also an option. For more details refer to [Multifactor Authentication in Auth0](/multifactor-authentication).
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-introduction.md]: @alemata `auth0` should be `Auth0`

File: articles/client-platforms/_includes/_mfa-enable.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The only action required in order to enable the MFA feature is open the [auth0 MFA settings]({$uiURL}/#/guardian) and enable the __Push Notifications__ option. You should see a page like that:
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-enable.md]: `{$uiURL}` should be `${uiURL}`

File: articles/client-platforms/_includes/_mfa-enable.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+![dashboard MFA with push notification enabled](/media/articles/mfa/guardian-push-enabled.png)
+
+After enabling it, hit the __save__ button and you're ready to test the Multi-factor authentication in your application.
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-enable.md]: @alemata For consistency please change `Multi-factor` to `Multifactor`. It's in several places in this doc.

File: articles/client-platforms/_includes/_mfa-login.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+There is no need to update the code we have created in [Login](/docs/quickstart/spa/react/01-login) tutorial. As soon you try to sign in, you should be prompted to setup an 2-Factor authenticator application (auth0 Guardian is the default).
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-login.md]: @alemata Change `setup an 2-Factor` to `setup a second factor`
Capitalize Auth0 (`auth0 Guardian` -> `Auth0 Guardian`)

File: articles/client-platforms/_includes/_mfa-login.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+![guardian screen](/media/articles/choose-mfa.png)
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-login.md]: @alemata Can't find this pic.

**Comment by alemata** [comment]
@mpaktiti updated!

**Pull request merged**